aea/test_tools/docker_image.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2021 Valory AG
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains testing utilities."""
+import logging
+import os
+import re
+import shutil
+import subprocess  # nosec
+import tempfile
+import time
+from abc import ABC, abstractmethod
+from typing import Dict, List, Optional
+
+import docker
```

Comment:
We don't want to make docker a dependency of the `open-aea`. I suggest a lazy import is utilised.